### DISCUSSION TITLE
Workspace chang with horizontal scroll

### DISCUSSION THREAD

**Discussion by miguelcollaco** [zen-browser/desktop#2282] [Ideas]
I believe it would be nice to add the horizontal scroll on the sidebar to change workspaces, just like arc.
It could be a setting, like vertical scroll on the sidebar to change tabs

**Comment by natyconnor** [comment]
Just want to bump this. When at my desk, I use an MX Master 3 mouse which has it's own horizontal scroll wheel. But it doesn't work to change workspaces in Zen, while it does on Arc. It's probably just an oversight since it's a pretty niche case to have a horizontal scroll wheel, but it's a game changer when using Arc

**Comment by JSGRD22** [comment]
This has been implemented now!

**Reply by aerscs** [reply]
Hey there! Any ways to disable it? Not coming from Arc but I have MX Master 3. I accidentally change my workspaces sometimes, so it would be really helpful to be able configure it somehow. Thank you guys for all of your work!

**Reply by sinbadthepoet** [reply]
Just want to second this, also mentioned it [here](https://github.com/zen-browser/desktop/pull/2828#issuecomment-2666676968)
